### PR TITLE
update BOM_1_0_2a11: reformatting

### PR DESCRIPTION
**Pull request by yatli**

### CODE REVIEW

File: BOM_1_0_2a11 [hunk 1]
```diff
@@ -1,70 +1,69 @@
...
-4   MIDIJACK            490-SDS-50                CP-2350                   U$7, U$8, U$9, U$10  DIN 5 jack also known as MIDI jack 
+4   MIDIJACK           490-SDS-50              CP-2350                   U$7, U$8, U$9, U$10                                                                                DIN 5 jack also known as MIDI jack 
         
-
 Switches:
 
 1   EG1218              SWITCH-SPDTKIT           SWITCH-SPDT_KIT           S1 
-4   3FT1D               3FT1D                    3FT                       BUTTON1, BUTTON2, BUTTON3, BUTTON4                                                                Mulitmec - 3FT1D                                                                                                                                                      
+4   3FT1D               3FT1D                    3FT                       BUTTON1, BUTTON2, BUTTON3, BUTTON4                                                               Mulitmec - 3FT1D                                                                                                                                                      
+??? SPDT Switch SWCH-08261
```
Comment (yatli) [inline BOM_1_0_2a11]: I don't see this in the PCB file. Replaced by EG1218?

Comment (jmamma) [reply BOM_1_0_2a11]: Hi Yatao,

I think you're looking at the older PCB design.

Just to be clear, there are two versions.

1_0_1a
1_0_2a12

The following BOM document refers to the 1_0_2a12 board.

Comment (yatli) [reply BOM_1_0_2a11]: I'm looking at megacommand_1_0_2a12.brd, and I can see sdcard pinout & EXP_L & EXP_R (which are absent in the earlier version right?), and S1 = EG1218

Comment (jmamma) [reply BOM_1_0_2a11]: That's correct

Comment (jmamma) [reply BOM_1_0_2a11]: I can only find reference to "SWCH-08261" on the old board, which might be a place holder I used before deciding on the EG1218

**Review by jmamma — COMMENTED** [review]
+1

**Pull request merged**